applications/nrf_desktop/README.rst

```diff
@@ -872,6 +718,164 @@ After building the application with or without specifying the :ref:`nrf_desktop_
 #. Connect to the mouse with an Android phone, a laptop, or any other Bluetooth Central.
    After the connection is established, you can use the mouse with the connected device.
 
+.. _nrf_desktop_porting_guide:
+
+Integrating your own hardware
```

Comment:
I am assuming this is a copy and not was changed

Reply:
Correct.